src/Sleep.js

```diff
@@ -17,42 +17,24 @@ class Sleep {
     return [...specificUser].splice(-7 * week, 7).map(day => day.date);
   }
 
-  returnAvgSleepHours() {
+  returnAvgSleepData(relevantProperty) {
+      let specificUser = this.findUser();
+      return Number((specificUser.reduce((totalHours, day) => {
+        totalHours += day[relevantProperty];
+        return totalHours;
+      }, 0) / specificUser.length).toFixed(2));
+    }
+
+  returnsleepData(date, relevantProperty) {
```

Comment:
change the casing here

Reply:
@MattTuring By casing do you mean returnsleepData should be returnSleepData?